doc/apps/s_client.pod

```diff
@@ -420,7 +420,7 @@ command for more information.
 send the protocol-specific message(s) to switch to TLS for communication.
 B<protocol> is a keyword for the intended protocol.  Currently, the only
 supported keywords are "smtp", "pop3", "imap", "ftp", "xmpp", "xmpp-server",
-"irc" and "postgres."
+"irc", "postgres", and "ldap".
```

Comment:
All docs I've seen have the punctuation outside the quotes. It would be very confusing to see it inside the quotes. It will be like the value to be put on command line includes punctuation.
I'm not a native speaker and I would not argue which way is correct according to english grammar. It will surely be confusing for users though in a technical documentation.